typescript/infra/src/utils/safe.ts

```diff
@@ -42,10 +42,12 @@ export async function createSafeTransaction(
   safeService: SafeApiKit.default,
   safeAddress: Address,
   safeTransactionData: MetaTransactionData[],
+  onlyCalls?: boolean,
 ): Promise<SafeTransaction> {
   const nextNonce = await safeService.getNextNonce(safeAddress);
   return safeSdk.createTransaction({
     safeTransactionData,
+    onlyCalls,
```

Comment:
So I understand correctly: we're still delegatecalling into the MultiSend contract, but the calls that are performed by the MultiSend contract can only be calls, not delegatecalls

Reply:
exactly